src/applications/appeals/testing/sc/utils/evidence.js

```diff
@@ -5,6 +5,7 @@ import { getIssueName, getSelected } from '../../../shared/utils/issues';
 export const hasVAEvidence = formData => formData?.[EVIDENCE_VA];
 export const hasPrivateEvidence = formData => formData?.[EVIDENCE_PRIVATE];
 export const hasOtherEvidence = formData => formData?.[EVIDENCE_OTHER];
+export const hasMstOption = formData => formData?.mstOption;
```

Comment:
Should `mstOption` be a constant like the others?

Reply:
Good idea!